### PR TITLE
Drop system cache for trie benchmarks

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: bin/node/testing/src/bench.rs [hunk 1]
```diff
@@ -88,6 +88,61 @@ impl BenchPair {
...
+
+	#[cfg(target_os = "macos")] {
+		log::trace!(target: "bench-logistics", "Clearing system cache...");
+		if let Err(err) = std::process::Command::new("purge").output() {
+			log::error!("purge error {:?}: ", err);
+			panic!("Could not clear system cache. Run under sudo?");
+		}
+		log::trace!(target: "bench-logistics", "Clearing system cache done!");
+	}
+
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs:143]:
```suggestion
```

File: bin/node/testing/src/bench.rs [hunk 2]
```diff
@@ -88,6 +88,61 @@ impl BenchPair {
 	}
 }
 
+/// Drop system cache.
+///
+/// Will panic if cache drop is impossbile.
+pub fn drop_system_cache() {
+
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs:95]:
```suggestion
```

File: bin/node/testing/src/bench.rs [hunk 3]
```diff
@@ -88,6 +88,61 @@ impl BenchPair {
...
+	#[cfg(target_os = "windows")] {
+		log::warn!(target: "bench-logistics",
+			"Clearing system cache on windows is not supported. Benchmark might totally be wrong.");
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs:98]:
```suggestion
		log::warn!(
			target: "bench-logistics",
			"Clearing system cache on windows is not supported. Benchmark might totally be wrong.",
		);
```

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**